src/scss/utilities/_avatar.scss

```diff
@@ -8,3 +8,14 @@
   flex: none;
   border-radius: 50%;
 }
+
+.avatar-xl,
```

Comment:
I think we do variations by using [data-size] attributes instead of class variations. So instead of 
`<a class="avatar-xl">` you'll use `<a class="avatar" data-size="xl"> `
See example here: https://github.com/GoogleChrome/web.dev/blob/main/src/scss/blocks/_feature-card.scss#L51